code/go/0chain.net/chaincore/chain/handler_integration_tests.go

```diff
@@ -51,15 +51,18 @@ func LatestFinalizedMagicBlockHandler(c Chainer) common.JSONResponderF {
 
 /*SetupHandlers sets up the necessary API end points */
 func SetupHandlers(c Chainer) {
-	hMap := handlersMap(c)
+	var hMap map[string]func(http.ResponseWriter, *http.Request)
 
 	if node.Self.Underlying().Type == node.NodeTypeMiner {
+		hMap := minerHandlersMap(c)
```

Comment:
Should not create new `hMap` here, the hMap will be empty. So let's use `hMap = minerHandlersMap(c)` instead